src/middlewared/middlewared/etc_files/rc.conf.py

```diff
@@ -223,7 +223,7 @@ def nfs_config(middleware, context):
     if nfs['v4']:
         yield 'nfsv4_server_enable="YES"'
 
-        if nfs['v4_krb'] and middleware.call_sync('datastore.query', 'directoryservice.kerberoskeytab'):
+        if nfs['v4_krb_enabled']:
```

Comment:
Somehow Ctrl+X, Ctrl+V at https://github.com/freenas/freenas/commit/52426935f1cf12494ded93b8f5bac3291f0a17a4 changed `or` to `and` causing this bug